airbyte-integrations/connectors/source-jira/source_jira/manifest.yaml

```diff
@@ -1417,8 +1417,8 @@ check:
     - issues
     - sprint_issues
 
-# This feature is currently not used, as only incremental streams with DatetimeBasedCursor are supported.
+# Defining a value here is difficult because the Jira API documentation mentions "REST API rate limits are not published because the computation logic is evolving continuously to maximize reliability and performance for customers" regarding the API budget. Hence, we will need to empirically validate the values that are set here.
 concurrency_level:
   type: ConcurrencyLevel
-  default_concurrency: "{{ config.get('num_workers', 3) }}"
+  default_concurrency: "{{ config.get('num_workers', 5) }}"
```

Comment:
Jira rate limiting is very unclear. [Their doc page](https://developer.atlassian.com/cloud/jira/platform/rate-limiting/) says `REST API rate limits are not published because the computation logic is evolving continuously to maximize reliability and performance for customers.` I've increase this a bit as I haven't seen any rate limiting issues with `num_workers = 5`. I'll monitor once we release in cloud